### ISSUE TITLE
send-http-level break list editor

### ISSUE THREAD

**Issue by mr-flibble** [duplicati/duplicati#3121]
Hello, if I paste --send-http-level=Success to config editor and then switch to list editor, problem occur:
I suspect some config parsing problem.
It should be simply reproducible

**Before adding --send-http-level=Success** (everything ok)

![image](https://user-images.githubusercontent.com/5197625/37850028-174e76ae-2eda-11e8-9138-e777ef0ee0c2.png)

**After adding --send-http-level=Success** (the settings are shifted)

![image](https://user-images.githubusercontent.com/5197625/37850051-2a972792-2eda-11e8-80e8-256b40169d5e.png)

After saving, config is just wrong:)

![image](https://user-images.githubusercontent.com/5197625/37850118-6097dc7e-2eda-11e8-8923-51d48ddd008d.png)

## Environment info

- **Duplicati version**: Duplicati - 2.0.2.13_canary_2017-11-22
- **Operating system**: Windows 10

**Comment by Pectojin** [comment]
This *looks* a bit like what I resolved in #2975, but not entirely.

I'm not having any luck replicating this on 2.0.3.2 when selecting the same option, which may mean it's been fixed by that or something else, but we'd have to verify that.

Do you have the option to test a similar setup on a newer Duplicati version?

**Comment by mr-flibble** [comment]
Hello, thanks for replicate advice!
I just upgraded to v2.0.3.2-2.0.3.2_canary_2018-03-16 on my pc and the problem remains.
I tried to replicate problem on another PC with 2.0.3.2 and there was no problem. 
I tried chrome anonymous windows on my PC and problem did not occur

**So in my Chrome I did ctrl +F5 and problem is gone....**
I'm so sorry, I should have thought... but it is still pretty weird

So thanks @Pectojin  :-)
